platform/kobo/disable-wifi.sh

```diff
@@ -9,9 +9,9 @@ ifconfig eth0 down
 
 # Some sleep in between may avoid system getting hung
 # (we test if a module is actually loaded to avoid unneeded sleeps)
-if lsmod | grep -q $WIFI_MODULE ; then
+if lsmod | grep -q "$WIFI_MODULE" ; then
```

Comment:
NiLuJe said he prefer both ;) I think it's better to keep it consistent to have the braces everywhere.